CHANGELOG.md

```diff
@@ -70,6 +70,10 @@ In addition, a number of improvements have been made to how the OLED display is
 - Fixed crash when using an external controller to control ModFX.
 - Fixed CC74 on the MPE master channel behaving like an expression event.
 
+### 3rd Party Device Integration
+
+- Hook support for adding signals to third-party devices starting with support for LED key highlighting and colours on the Lumi Keys Studio Edition.
+
```

Comment:
Under User Interface above we have a point on the LUMI Keys.

"- Added support for LUMI Keys SYSEX protocol protocol. When hosting a LUMI Keys keyboard, the current scale will automatically be set on the keyboard."

I like the 3rd Party Device Integration section though. I would suggest removing the bullet from the User Interface section and maybe combine the wording from both. 

E.g. "Added support for third-party devices SYSEX protocol integration starting with the LUMI Keys Studio Edition. When hosting a LUMI Keys keyboard, the current scale and key colours will be automatically set."

Reply:
Ah alright. Missed that. I'll remove my CHANGELOG.md addition.